src/main/java/com/frc/codex/clients/companieshouse/FilingFormat.java

```diff
@@ -0,0 +1,30 @@
+package com.frc.codex.clients.companieshouse;
+
+public enum FilingFormat {
+	XML ("xhtml", "application/xml", 1),
```

Comment:
When is the XML format used?

Reply:
Wish I would have left a comment, but I'm guessing I originally added this because the `application/xml` content type was encountered and associated with an inline XBRL filing earlier on. I could add some info logs to get a sense for how often, if ever, it shows up with recent filings.